adapter-excel/src/test/java/ch/prevo/open/node/adapter/excel/ExcelAssertions.java

```diff
@@ -53,4 +56,20 @@ public static void assertRow(String filename, String sheet, int rowIndex, Object
         }
     }
 
+    public static void assertRowComments(String filename, String sheet, int rowIndex, String... expectedCommentsValues) throws IOException, InvalidFormatException {
+        try (final Workbook workbook = WorkbookFactory.create(new File(filename), null, true)) {
+            final Row row = workbook.getSheet(sheet).getRow(rowIndex);
+            List<String> commentsInRow = collectAllCommentsInRow(row);
+            assertThat(commentsInRow).hasSize(expectedCommentsValues.length);
```

Comment:
Is this check really required? The output is not as useful as the one from the following assert.